ansible/roles/gelato-ha-installer/tasks/main.yml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2021 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Note: It is assumed that the Kubernetes master node has all the prereqs for running Multicloud services. 
+#       If any dependency is found to be missing, it will be opened and fixed as issues
+
+---
+- name: create gelato-ha work directory if it doesn't exist
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: 0755
+  with_items:
+    - "{{ gelato_work_dir }}"
+    - "{{ gelato_tarball_dir }}"
+
+- name: include scenarios/release.yml when installing from release
+  include: scenarios/release.yml
+  when: install_gelato_ha_from == "release" or install_gelato_ha_from == "container"
+
+
+- name: Execute the kubernetes manifests.
+  shell: "{{ item }}"
+  args:
+    chdir: "{{ gelato_work_dir }}/multi-cloud/K8s"
+  with_items:
+    - bash "k8s-mongo.sh"
+    - bash "k8s-deploy.sh"
+  become_user: ubuntu
```

Comment:
Pls suggest how to handle the become_user:<username>

Reply:
should we take it from the group_vars/gelato-ha.yml

Reply:
You may need to document user creation, or handle it in a script.

Reply:
Does the HA only supported in Ubuntu? You can document the OS & Version that you tested

Reply:
ok , only ubuntu for now.